manifest_structure.md

```diff
@@ -0,0 +1,31 @@
+Manifest structure
+==================
+
+The structure for manifests are created with the following requirements:
+
+1. When adding or refactoring the baseline, the number of files needed to edit
+   should be as low as possible.
+2. The number of manifests files needed per supported hardware target should
+   be as low as possible.
+
+Structure
+---------
+
+![IMAGE OF MANIFEST STRUCTURE](manifest_structure.svg)
+
+In the above image, `Base` is the manifest file containing mappings to the git
+repositories which are required by every PELUX build (i.e. `PELUX-base.xml`).
+`Qt` is the manifest file that contain mappings to the git repositories required
+in every PELUX QtAS build which is not already covered in `Base`.
+
+In contrast to `Base` and `Qt` there multiple instances of `Bsp` and
+`Bsp + QtAs`. However, each such instance holds mappings to git repositories
+required by specific hardware target. This means that there is no one "Bsp"
+or "Bsp + QtAS" manifest file, but one file each per hardware target. An example
+of this are `pelux-intel.xml` and `pelux-intel-qt.xml`this which represent
+specific implementations of `Bsp` and `Bsp + QtAS` for the intel hardware target.
+
+Using this this structure minimizes the number of manifest files in the
+repository. The amount of files can be calculated by the formula `2 * n + 2`
```

Comment:
IMHO it's enough to state that the purpose is to reduce amount of files etc. (as you already do).